Makefile

```diff
@@ -1,23 +1,46 @@
-all: output/rootfs.cpio.xz output/bzImage
+TARGETS := output/rootfs.cpio.xz output/bzImage
+SOURCES := Dockerfile config/buildroot config/kernel config/user bin/build_dhyve
```

Comment:
hmm.. will this trigger a rebuild if something changes in the rootfs directory too?